lib/screens/home.dart

```diff
@@ -104,6 +104,74 @@ class HomeState extends State<Home> with SingleTickerProviderStateMixin {
     }
   }
 
+  Future<void> _writePrivateData() async {
+    setState(() {
+      // Begin loading.
+      _isLoading = true;
+    });
+
+    final appName = await getAppName();
+
+    // final filePath = '$appName/data/test-101.ttl';
+    // final fileContent = 'This is for testing writePod.';
+
+    final filePath = '$appName/data/test-102.ttl';
```

Comment:
From app developer point of view, would it work to assume the basename is $appname/data. Then I simply have a filename `test-102.ttl` as the file path. That seems a simpler arrangement for the app developer.

Reply:
Sounds good. A function to get the data path is provided in `solidpod` which is basically `$appName/data`.